hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/RandomKeyReader.java

```diff
@@ -0,0 +1,104 @@
+package org.apache.hadoop.ozone.freon;
+
+import com.codahale.metrics.Timer;
+import kotlin.Pair;
```

Comment:
Please use org.apache.commons.lang3.tuple.Pair instead of Kotlin's Pair, it is a bit different, instead of a constructor you can create one with the Pair.of(left, right) static method, and instead of getFirst and getSecond it has getLeft and getRight respectively.